### DISCUSSION TITLE
Bug when renaming the collections

### DISCUSSION THREAD

**Discussion by VJBB23** [dani-garcia/vaultwarden#3712] [General]
We unfortunately have a bug when renaming the collections, if a collection have child collections and I rename the chairing one the subcategories lose the link. This is annoying when there are many subcategories...
![after-rename](https://github.com/dani-garcia/vaultwarden/assets/125476393/d5d3e2da-2aab-449a-a5f2-ee14147b56e5)
![before-rename](https://github.com/dani-garcia/vaultwarden/assets/125476393/72cffa0f-5bb6-49e7-8ebb-e25e10c93e54)

**Comment by BlackDex** [comment]
This is not something we can change as this is a client/web-vault issue.
Vaultwarden (backend server) doesn't even know what the name of the collection is, or folder for that matter, because those are almost the same.

The client's search for the `/` and nest them. On the server side there is no way to generate a connection.

**Comment by MariaPaypoint** [comment]
Issue in the web client repo is actual https://github.com/bitwarden/clients/issues/6217
